File: Beginner/Uri1013.java
package Beginner;

import java.util.Scanner;

public class Uri1013 {

	public static void main(String[] args) {
		
		Scanner sc = new Scanner(System.in);
		int a = sc.nextInt();
		int b = sc.nextInt();
		int c = sc.nextInt();
		
		int maiorAB = (a + b + Math.abs(a - b))/2;
		int maiorABC = (maiorAB + c + (Math.abs(maiorAB - c)))/2;
		
		if (maiorABC == a) {
			System.out.println(a + " eh o maior");
		}
		
		else if (maiorABC == b) {
			System.out.println(b + " eh o maior");
		}
		else {
			System.out.println(c + " eh o maior");
		}
	}

}

File: src/beginner/Uri1013.py
linha1 = input().split(" ")
a , b , c = linha1

maiorAB = (int(a) + int(b) + abs(int(a) - int(b))) / 2
resultado = (int(maiorAB) + int(c) + abs(int(maiorAB) - int(c))) / 2

print("%d eh o maior" %resultado)